main.py

```diff
@@ -1,10 +1,54 @@
+import json
+
 import init_django_orm  # noqa: F401
 
+from django.utils.timezone import now
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data["race"]
+        race_name = race_data["name"]
+        race_description = race_data.get("description", "")
+        race, _ = Race.objects.get_or_create(
+            name=race_name,
+            defaults={"description": race_description}
+        )
+
+        guild_data = player_data.get("guild", None)
+        if guild_data:
+            guild_name = guild_data["name"]
+            guild_description = guild_data.get("description", None)
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_name,
+                defaults={"description": guild_description}
+            )
+        else:
+            guild = None
```

Comment:
you can move this above line 24 and so you will not need this else condition at all